### DISCUSSION TITLE
[@types/shelljs] change dependency from types/glob to glob

### DISCUSSION THREAD

**Discussion by jpshack-at-palomar** [DefinitelyTyped/DefinitelyTyped#69867] [Issues with a @types package]
```
$ npm ls @types/glob
└─┬ @types/shelljs@0.8.15
  └── @types/glob@7.2.0
```

Leads to issues such as:   https://stackoverflow.com/questions/75890950

```
node_modules/@types/glob/index.d.ts:29:42 - error TS2694: Namespace '"/opt/workspace/personal/kioskify-backend/functions/node_modules/minimatch/dist/cjs/index"' has no exported member 'IOptions'.

29     interface IOptions extends minimatch.IOptions {
                                            ~~~~~~~~

node_modules/@types/glob/index.d.ts:75:30 - error TS2724: '"/opt/workspace/personal/kioskify-backend/functions/node_modules/minimatch/dist/cjs/index"' has no exported member named 'IMinimatch'. Did you mean 'Minimatch'?

75         minimatch: minimatch.IMinimatch;
                                ~~~~~~~~~~

node_modules/@types/rimraf/index.d.ts:33:21 - error TS2694: Namespace '"/opt/workspace/personal/kioskify-backend/functions/node_modules/glob/dist/mjs/index"' has no exported member 'IOptions'.

33         glob?: glob.IOptions | false | undefined;
                       ~~~~~~~~


Found 3 errors in 2 files.

Errors  Files
     2  node_modules/@types/glob/index.d.ts:29
     1  node_modules/@types/rimraf/index.d.ts:33
```

Also reported in the shelljs project here: https://github.com/shelljs/shelljs/issues/1148

Similar work done for other packages such as in: #65258.
See also:
 - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67102
 - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64806
 - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64810

[a comment by an automated account is not shown]

**Comment by jpshack-at-palomar** [comment]
See also the this report #68277 (I had missed this because I was looking in issues not discussions. Sorry for that.)

**Reply by nikeee** [reply]
It seems like the right solution to switch to `glob`. I'm currently limited in my capacity, but we're open for PRs!

**Comment by jayeshtemp** [comment]
I am facing this issue as well, do we have any temporary workaround to avoid these errors?

**Reply by jpshack-at-palomar** [reply]
My workaround is here: https://stackoverflow.com/questions/75890950/node-modules-minimatch-dist-cjs-index-has-no-exported-member-ioptions/78649918#78649918
